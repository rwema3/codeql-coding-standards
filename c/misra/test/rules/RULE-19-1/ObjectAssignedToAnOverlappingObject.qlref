rules/RULE-19-1/ObjectAssignedToAnOverlappingObject.ql