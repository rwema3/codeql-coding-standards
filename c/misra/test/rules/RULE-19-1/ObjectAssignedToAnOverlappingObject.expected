| test.c:55:3:55:18 | ... = ... | An object $@ assigned to overlapping object $@. | test.c:55:9:55:10 | m2 | m2 | test.c:55:17:55:18 | m1 | m1 |
